hack/example-images.json

```diff
@@ -0,0 +1,4 @@
+{
+    "cloudControllerManagerAWS": "registry.svc.ci.openshift.org/openshift:aws-cloud-controller-manager",
+    "cloudControllerManagerOpenStack": "registry.svc.ci.openshift.org/openshift:openstack-cloud-controller-manager"
```

Comment:
we should probably fix these up before committing. no reason to use the old url

Reply:
Right, let me adjust that. CI is now passing